org.eclipse.jdt.core.tests.compiler/src/org/eclipse/jdt/core/tests/compiler/regression/CompilerInvocationTests.java

```diff
@@ -158,6 +158,9 @@ public void acceptResult(CompilationResult compilationResult) {
 	}
 
 	static String taskTagsAsCutAndPaste(CategorizedProblem tasks[]) {
+		if (tasks == null) {
+			return "";
+		}
```

Comment:
Curious - what necessitates this ?

Reply:
The tests expect to run with the **platform** started (not just as usual JUnit tests), and if platform is not there, fail with obscure NPE's instead of failing with assertions about missing tasks. I've fixed it here to see actual issue what the test wanted to test.